src/ensembl/production/metadata/client_examples.py

```diff
@@ -149,6 +151,12 @@ def get_top_level_statistics(stub):
     print(releases1)
 
 
+def get_datasets_list_by_uuid(stub):
+    request1 = DatasetsRequest(genome_uuid='a73351f7-93e7-11ec-a39d-005056b38ce3', release_version='2020')
```

Comment:
I get this error when I try to run these examples:

```
TypeError: '2020' has type str, but expected one of: int, long
```

Reply:
thanks for pointing this out, it should be numeric 
```
request1 = DatasetsRequest(genome_uuid='a73351f7-93e7-11ec-a39d-005056b38ce3', release_version=2020) 
```